lib/marmite/models/record.rb

```diff
@@ -2,6 +2,8 @@
 
 class Record < ActiveRecord::Base
 
+  FORMATS_TO_ALWAYS_RECREATE = %w[structural_ark combined_ark iiif_presentation]
```

Comment:
Why are structural_ark and combined_ark always recreated? I don't remember. :/

Reply:
i have no idea

Reply:
Is this why? https://github.com/upenn-libraries/marmite/blob/e62c3c133327bca0211f6d599627844ad81b56a0/application.rb#L476

Reply:
https://github.com/upenn-libraries/marmite/commit/33035a4b86de22a51016a44eeddc09736ce4d98c is not helpful

Reply:
yes - i guess the proper answer to your original question was "because they were before" - _why_ they were before, i don't know

Reply:
Good enough for me. We can leave it as is.